labs/a3.md

```diff
@@ -215,21 +222,38 @@ This is a very common scheme and is totally a scheme you could use if
 you were installing Arch on your own computer (except that you'll
 hopefully have more than 5GiB to work with)!
 
+**GRADESCOPE**: Give a brief and basic explanation of the purpose of having (1)
+the EFI system partition and (2) the swap partition. These explanations can be
+as short as one sentence, just try to get a general sense of why we have these
+partitions.
+
 ### create partitions
 
-We will use `fdisk` to create our partitions. Run `fdisk /dev/sda` to start.
+We will use `fdisk` to create our partitions. Before we start the actual
```

Comment:
Perhaps this would be a good place to add a link to more resources about `fdisk`

Reply:
Added link to the fdisk article on the Arch Wiki.